src/dotnet/UI.Blazor/Components/SideNav/side-nav.css

```diff
@@ -0,0 +1,56 @@
+.side-nav {
+    @apply fixed md:static;
+    @apply h-full w-full md:w-64 lg:w-80;
+    @apply md:!transform-none;
+    @apply text-primary bg-primary;
+    @apply top-0 bottom-0;
+}
+
+.side-nav-right {
+    transform: translate3d(100%, 0, 0);
+    right: 0;
+    top: 49px;
+    @apply z-40;
+}
+
+.side-nav-left {
+    transform: translate3d(-100%, 0, 0);
+    left: 0;
+    @apply z-50;
+}
+
+.side-nav-drawer {
+    position: fixed;
```

Comment:
Kinda fine, but I would try to use tailwind classes whenever it's possible for the sake of consistency.